src/com/rcv/CastVoteRecord.java

```diff
@@ -42,19 +43,25 @@ public void addRoundDescription(String description, int round) {
     return mSortedRankings;
   }
 
-  public CastVoteRecord(String ballotID, List<ContestRanking> rankings, List<String> fullCVRData) {
+  public CastVoteRecord(String source, String ballotID, List<ContestRanking> rankings, List<String> fullCVRData) {
+    mSource = source;
     mBallotID = ballotID;
     mRankings = rankings;
     mFullCVRData = fullCVRData;
   }
 
   String getAuditString() {
     StringBuilder auditStringBuilder = new StringBuilder();
+    auditStringBuilder.append("[CVR Source] ");
+    auditStringBuilder.append(mSource);
+    auditStringBuilder.append(" [Ballot ID] ");
     auditStringBuilder.append(mBallotID);
-    auditStringBuilder.append(":");
+    auditStringBuilder.append(" [Round by Round Report] ");
     for(Integer round : mDescriptionsByRound.keySet()) {
       auditStringBuilder.append(mDescriptionsByRound.get(round));
```

Comment:
Is there a delimiter between rounds?

Reply:
Pipes appear between round reports per Caleb's email.  I'll add a leading pipe for the first round so they all look the same: 

Mon Jan 08 08:42:14 PST 2018 INFO: [CVR Source] test/2015-portland-mayor-cvr-candidate-codes.xlsx [Ballot ID] 2015-portland-mayor-cvr-candidate-codes.xlsx(77) [Round by Round Report] 1|CEB|2|CEB|3|CEB|4|CEB|5|CEB|6|CEB|7|CEB|8|CEB|9|CEB|10|CEB|11|CEB|12|CEB|13|CEB|14|NMM| [Raw Data] [77.0, D Cliff and GDI, D Cliff and GDI, CEB , HAH , JME , CLV , NMM , JLL , MSM , JCD , RAD , RCC , JR , EKS , undervote, DAM , PGB ]